src/Reports/SubsiteReportWrapper.php

```diff
@@ -59,9 +60,13 @@ public function columns()
     /**
      * @param arary $params
      * @return void
+     * @deprecated 2.8.0 Will be removed without equivalent functionality to replace it
      */
     public function beforeQuery($params)
     {
+        Deprecation::withNoReplacement(function () {
+            Deprecation::notice('2.8.0', 'Will be removed without equivalent functionality to replace it');
+        });
```

Comment:
Refer to https://github.com/silverstripe/silverstripe-admin/pull/1453#discussion_r1098134905 for this and the others in this PR